src/sql/ConfigEditor/ConfigSelect.tsx

```diff
@@ -70,7 +70,6 @@ export function ConfigSelect(props: ConfigSelectProps) {
       allowCustomValue={props.allowCustomValue}
       autoFocus={props.autoFocus}
       backspaceRemovesValue={props.backspaceRemovesValue}
-      className={props.className}
```

Comment:
I removed passing the `className` prop down as the current behaviour is already overriding the className (see `commonProps` on line 43).

Reply:
Should we still be passing `'width-30'` as the className?

Reply:
It's still being passed by commonProps.